src/brainglobe_napari/napari_atlas_representation.py

```diff
@@ -32,20 +32,34 @@ def add_to_viewer(self):
             name=f"{self.bg_atlas.atlas_name}_annotation",
         )
 
-        root_mesh = self.bg_atlas.mesh_from_structure("root")
-        self._add_mesh(root_mesh, name=f"{self.bg_atlas.atlas_name}_mesh")
+    def add_structure_to_viewer(self, structure_name: str):
+        """Adds the mesh of a structure to the viewer
 
-    def _add_mesh(self, mesh: Mesh, name: str = None):
+        structure_name: the id or acronym of the structure.
+        """
+        mesh = self.bg_atlas.mesh_from_structure(structure_name)
+        color = self.bg_atlas.structures[structure_name]["rgb_triplet"]
```

Comment:
On my machine, I don't see any colors, nor can I change them within napari.

Reply:
Partially outsourced to #46